.github/workflows/deploy.yml

```diff
@@ -0,0 +1,45 @@
+name: Deploy
+
+on:
+  workflow_run:
+    workflows:
+      - Test
```

Comment:
should this be "Deploy"?

Reply:
@sleepylee As I set as `Test` because I'll trigger this workflow after `Test` complete ✌️